style.css

```diff
@@ -9,14 +10,19 @@ body {
   --black: #292524;
   --greyOne: #44403c;
   --greyTwo: #57534e;
+  --greyFour: #e2e8f0;
   --teal1: #115e59;
   --teal2: #2dd4bf;
   --teal3: #f0fdfa;
+  --red1: #dc2626;
   --alert-green: #d9f99d;
   --alert-blue: #bfdbfe;
   --alert-yellow: #fde68a;
   --alert-red: #fecaca;
   --white1: #fafaf9;
+  --yellow2: #fbbf24;
+  --orange2: #fb923c;
+  --blue2: #60a5fa;
```

Comment:
1. Naming conventions for the colors -- based on their shade.
2. move them :root instead of `body`

Reply:
I did all the necessary changes here @supminn , can you please check it out.

Reply:
The changes look fine to me.